docs/serverless/08-08-logging-in-to-private-packages-registry.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Logging in to private package registry using credentials from Secret
+type: Tutorials
+---
+
+This tutorial shows how you can login in to private package registry through define credentials in Secret. 
+
+## Steps
+
+### Create a Secret 
+
```

Comment:
Create a Secret CR for your Node.js or Python Functions. You can also create one combined Secret CR for both runtimes.

Reply:
Done